overlord/devicestate/handlers_install.go

```diff
@@ -92,7 +92,12 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 	}
 
 	// configure the run system
-	if err := sysconfigConfigureRunSystem(&sysconfig.Options{}); err != nil {
+	opts := &sysconfig.Options{}
+	cloudCfg := filepath.Join(dirs.RunMnt, "ubuntu-seed/cloud.cfg.d")
+	if osutil.IsDirectory(cloudCfg) && deviceCtx.Model().Grade() == asserts.ModelDangerous {
```

Comment:
this probably merits a comment spelling out the rule enforced here atm

later we might want to push down this check into configureRunSystem itself but need to see how that evolves

Reply:
I need to think about the spread test, added a comment for now. Three is no real support for ubuntu-image yet so a spread test would unpack/repack the generated image, boot it etc. It's a bit involved but maybe I have not thought hard enough about it.

Reply:
@mvo5 an easy thing to do for this is to add a basic cloud.cfg to the default image we build and test with on spread, then have a spread test check that we got the files copied into the running system